### DISCUSSION TITLE
Menu icons missing after updating to version 3.0.4

### DISCUSSION THREAD

**Discussion by SaberCrear** [templaza/astroid-framework#465] [Q&A] [answered]
Hello, I have verified that with the latest update, the icons on the menu buttons disappear. Did it happen to someone else?
Thanks

![photo_2023-11-12_13-11-27](https://github.com/templaza/astroid-framework/assets/123734392/1d6c61da-6149-4484-97f0-718b33525e19)

**Comment by sonvnn** [comment, marked as answer]
This issue was fixed in 3.0.5. Please update Astroid. Thanks

**Reply by SaberCrear** [reply]
OK. Thank you very much

**Comment by SaberCrear** [comment]
Hello again. When updating to version 3.0.5 you can already see the icons, but now this new version has broken all the easy Discuss links. All discussions lead to the 404 error page. Can you help me?

**Reply by sonvnn** [reply]
It's very strange. May you send me the link via sonlv@templaza.com

**Reply by SaberCrear** [reply]
I have fix it. Thank you so much
